mysql-test/suite/rpl/t/percona_bug_ps7599.test

```diff
@@ -0,0 +1,20 @@
+#
```

Comment:
IMO, using JIRA number in the testcase name is not very intuitive and will make the person to actually look into the test while categorizing the failure in case the test fails in future.

Can we rename the testcase to `rpl_update_partitioned_table_with_compressed_column.test` or something like that?